ml-testing-toolkit/chart-backend/values.yaml

```diff
@@ -352,7 +352,6 @@ resources: {}
 ## Persist data to a persistent volume
 persistence:
```

Comment:
This config looks like its related to MySQL/Percona.

I suspect removing the backend dependency will not be straight forward here as a result. We will need to double check this.

Reply:
reverted